indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1995,25 +1993,8 @@ def firstPage(self, c, doc):
 
         # QRCode (Version 6 with error correction L can contain up to 106 bytes)
         height -= 6*cm
-        qr = QRCode(
-            version=6,
-            error_correction=constants.ERROR_CORRECT_M,
-            box_size=4,
-            border=1
-        )
-        config = Config.getInstance()
-        baseURL = config.getBaseSecureURL() if config.getBaseSecureURL() else config.getBaseURL()
-        qr_data = {"registrant_id": self._registration.id,
-                   "checkin_secret": self._registration.ticket_uuid,
-                   "event_id": self._conf.getId(),
-                   "server_url": baseURL
-                   }
-        json_qr_data = json.dumps(qr_data)
-        qr.add_data(json_qr_data)
-        qr.make(fit=True)
-        qr_img = qr.make_image()
-
-        c.drawImage(ImageReader(qr_img._img), width, height, height=4*cm,
+        qr_img = generate_ticket_qr_code(self._registration)
+        c.drawImage(ImageReader(qr_img), width, height, height=4*cm,
                     width=4*cm)
```

Comment:
OK, not an accident. I basically fixed legacy code so that it would work with the new version. Anyway, i'll just revert that change.